.github/workflows/build_whl_and_publish.yaml

```
@@ -0,0 +1,40 @@
+name: Build nightly wheels and publish to PyTorch Index
+
+on:
+  push:
```

```suggestion
  pull_request:
  push:
```
@joecummings to test this workflow you can also enable it on pull request

---

Yep, did this on past commits :)